Jenkinsfile

```diff
@@ -679,6 +673,27 @@ pipeline {
             }
         }
 
+        stage("Full Tests gfx1030"){
+            when { expression { params.MIOPENNAVI21 && !params.DISABLE_ALL_STAGES } }
+            environment{
+                WORKAROUND_iGemm_936 = " MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0"
+            }
+            parallel{
+                stage('Fp32 OpenCL All gfx1030') {
+                        agent{ label rocmnode("navi21") }
+                        steps{
+                            buildHipClangJobAndReboot(compiler: 'g++', setup_flags: Full_test, build_env: extra_log_env, gpu_arch: "gfx1030")
+                        }
+                    }
+                stage('Fp32 Hip All Install gfx1030') {
+                        agent{ label rocmnode("navi21") }
+                        steps{
+                            buildHipClangJobAndReboot(setup_flags: Full_test, build_env: WORKAROUND_iGemm_936, build_install: "true", gpu_arch: "gfx1030")
```

Comment:
WORKAROUND_iGemm_936 is for HALF and not needed for FP32 stages. This came from #147. I'll fix this soon.